<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rulesDialog</class>
 <widget class="QDialog" name="rulesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Topology Rule Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Current Rules</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="mLayer1Box">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>No layer</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mRuleBox">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mLayer2Box">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>No layer</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="mToleranceLabel">
         <property name="text">
          <string>Tolerance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="mToleranceBox"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="mAddTestButton">
         <property name="text">
          <string>Add New Rule</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mDeleteTestButton">
         <property name="text">
          <string>Delete Rule</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableWidget" name="mRulesTable">
     <column>
      <property name="text">
       <string>Rule</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Layer #1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Layer #2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tolerance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Layer1ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Layer2ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>rulesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>rulesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
